docs/tutorial/data.md

```diff
@@ -536,28 +536,28 @@ Oh no! What went wrong?
 To find out what went wrong, we need to use `firstdiff.py`. This tells us where our ROM starts to differ:
 ```
 $ ./firstdiff.py
-First difference at ROM addr 0x144F4, gDmaDataTable (RAM 0x80016DA0, ROM 0x12F70, build/asm/dmadata.o)
+First difference at ROM addr 0x144F4, gDmaDataTable (RAM 0x80016DA0, ROM 0x12F70, build/gc-eu-mq-dbg/asm/dmadata.o)
 Bytes: 00:E3:F9:D0 vs 00:E3:F9:E0
-Instruction difference at ROM addr 0xE3ED48, En_Jj_InitVars (RAM 0x80A88140, ROM 0xE3ED10, build/src/overlays/actors/ovl_En_Jj/z_en_jj.o)
+Instruction difference at ROM addr 0xE3ED48, En_Jj_InitVars (RAM 0x80A88140, ROM 0xE3ED10, build/gc-eu-mq-dbg/src/overlays/actors/ovl_En_Jj/z_en_jj.o)
 Bytes: 40:00:00:00 vs 00:F0:00:00
```

Comment:
I appreciate wanting to update the firstdiff.py output but it's anachronistic here, the output of the tool changed beyond just the build/ path since this was copypasted. I would just leave what the tutorial currently has for first_diff.py output (same below)

(changes in the tutorial about spec segments are fine, I mean specifically the firstdiff.py stuff)

Reply:
Okay, I reverted the output